dev/standalone-community/docker-compose-ui.yml

```diff
@@ -5,4 +5,4 @@ services:
     environment:
       - "API_PROXY_HOST=api"
       - "API_PROXY_PORT=8000"
-      - "DEPLOYMENT_MODE=${COMPOSE_PROFILE}"
```

Comment:
shouldn't this be `DEPLOYMENT_MODE=standalone-community` ?

Reply:
Actually, we have only 2 deployment modes AFAIK: standalone and insights, I think we don't want to add a new one, is that useful? or can we stick with `standalone` here?

Reply:
Maybe, but all the the other UI files are using a variable:

```
[jtanner@p1 galaxy_ng]$ find dev -type f -name "*ui.yaml" | xargs fgrep -H DEPLOYMENT_MODE=
dev/insights/docker-compose-ui.yaml:      - "DEPLOYMENT_MODE=${COMPOSE_PROFILE}"
dev/standalone-keycloak/docker-compose-ui.yaml:      - "DEPLOYMENT_MODE=${COMPOSE_PROFILE}"
dev/standalone-ldap/docker-compose-ui.yaml:      - "DEPLOYMENT_MODE=${COMPOSE_PROFILE}"
dev/standalone/docker-compose-ui.yaml:      - "DEPLOYMENT_MODE=${COMPOSE_PROFILE}"
dev/standalone-community/docker-compose-ui.yaml:      - "DEPLOYMENT_MODE=${COMPOSE_PROFILE}"
```

Reply:
I hardcoded it to standalone. We may need to revise later when v1 support is added depending on the capability of feature flags.